pkg/runtime/optimize/dml/select.go

```diff
@@ -51,6 +52,62 @@ func init() {
 
 func optimizeSelect(ctx context.Context, o *optimize.Optimizer) (proto.Plan, error) {
 	stmt := o.Stmt.(*ast.SelectStatement)
+	enableLocalMathComputation := ctx.Value(proto.ContextKeyEnableLocalComputation{}).(bool)
+	if enableLocalMathComputation && len(stmt.From) == 0 {
+		isLocalFlag := true
+		var columnList []string
+		var valueList []proto.Value
+		for i := range stmt.Select {
+			switch stmt.Select[i].(type) {
+			case *ast.SelectElementExpr:
+				var nodeInner *ast.PredicateExpressionNode
+				calculateNode := stmt.Select[i].(*ast.SelectElementExpr).Expression()
```

Comment:
Declare a variable instead, for example:
```go
switch t := stmt.Select[i].(type) {
case *ast.SelectElementExpr:
      _ = t.Expression()
    // ...
}

```

Reply:
Processed